_drafts/contribute-to-ansible.markdown

```diff
@@ -1,5 +1,9 @@
-# How to contribute on Ansible project
-
+---
+layout: post
+title:  "How to contribute to ansible project"
+date:   2016-12-6 12:55:28 +0100
+categories: ansible contribute version git fork
+---
 ## Introduction
 
 Because we find Ansible, just awsome, we would like to share our experience about contributing on it. 
```

Comment:
Rather than considering, Ansible  needs contributorS to continue to improve its quality and stability every days.